components/LogoutContainer.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react'
 import { useRouter } from 'next/router'
-import { useLogoutMutation } from '../graphql'
+import { signOut } from 'next-auth/react'
 
 export const LogoutContainer: React.FC = ({ children }) => {
   const router = useRouter()
-  const [logoutUser] = useLogoutMutation({
-    update(cache) {
-      cache.modify({
```

Comment:
Expose the cache so we can modify without having to execute a query. There's probably a reason for setting the `lessonStatus` to an empty array.